### ISSUE TITLE
Add support for EL9

### ISSUE THREAD

**Issue by leeiitb** [moby/moby#43290]
Hi,

Kindly add support for EL9.
RHEL 9 is in Beta, and CentOS Stream 9 is active.

**Comment by linghengqian** [comment]
Supplementary to CentOS 9 Stream only. Pointing to https://github.com/docker/roadmap/issues/303 .

**Comment by jrgruher** [comment]
I am also looking for this support.

**Comment by perriernical** [comment]
I am also looking for this support.

**Comment by linghengqian** [comment]
This issue has been resolved as it is Fedora in nature and should be closed.

**Comment by thaJeztah** [comment]
Yes, packages for CentOS stream 9 are available on download.docker.com; https://download.docker.com/linux/centos/9/
